README.md

```diff
@@ -228,88 +208,9 @@ send.mime.define({
 })
 
 var server = http.createServer(function onRequest (req, res) {
-  send(req, parseUrl(req).pathname, { root: '/www/public' })
-    .pipe(res)
-})
-
-server.listen(3000)
-```
-
-### Custom directory index view
```

Comment:
is the custom directory index implementation gone?

Reply:
It is based on the event API and we might design how to implement it later.